src/js/api/config.js

```diff
@@ -56,6 +56,7 @@ const Defaults = {
         rewind: 'Rewind 10 Seconds',
         settings: 'Settings',
         stop: 'Stop',
+        share: 'Share',
```

Comment:
We might want to make `sharing` a block within `localization` (similar to `errors`) since there are plans to add more fields and customizations to sharing